### DISCUSSION TITLE
Problem with touchscreen on Windows (Surface Pro 7)

### DISCUSSION THREAD

**Discussion by janvinas** [xournalpp/xournalpp#4013] [Q&A]
I've just installed Xournal++ for Windows on my Microsoft Surface Pro 7 (Windows 10 Home 21H1 19043.1645), and I've noticed that the touchscreen has an incredibly strange behaviour. 
- On the main writing page, trying to scroll with one finger makes the page go totally crazy and start randomly scrolling in both directions. However, scrolling with two fingers works fine (of course, also modifying zoom level if enabled in the configuration page).
- On the configuration window, the behaviour is even weirder: I can perfectly scroll the right pane with touch if it doesn't fit in the window (for example the View section). But if I select an entry on the left menu using the touch screen, then the right panel becomes unresponsive for touch (recovers again for touch if I interact with the window with the mouse pointer or the pen)

It's a pity because otherwise it's a very good app and I would love using it on a daily basis for taking notes. Do you have any ideas on why is this happening? Is it a bug or some weird configuration that I could change? I'm attaching some screenshots of my config to further help diagnose the problem.

Thanks.

![image](https://user-images.githubusercontent.com/35854198/167252048-1336429b-0066-40b5-8316-2431dbbcc19b.png)

![image](https://user-images.githubusercontent.com/35854198/167252071-0c29cd46-541b-4b63-9669-469efe1e8072.png)

**Comment by danemtsov** [comment]
If you are using the nightly build, the scrolling issue can be worked around by disabling inertial scrolling in  _Preferences > Touchscreen_.

The other touch issue is a GTK bug. It also happens e.g. when you miss a tool button and touch the toolbar instead – then the whole window becomes unresponsive until you click somewhere using mouse or stylus.

**Reply by janvinas** [reply]
Thanks for your quick response.

I have installed the latest nightly build and disabling GTK inertial scroll does indeed fix the problem. However, I'm now getting an ink rendering glitch. Is it possible to disable GTK inertial scroll on the latest stable build?

![image](https://user-images.githubusercontent.com/35854198/167255029-a516aec5-0756-4955-9524-5c0ee82b8f6f.png)

**Reply by danemtsov** [reply]
Unfortunately this feature is not in stable yet.

The rendering glitch looks similar to #4011, but I can't reproduce it. Have you tried disabling scrollbar fadeout (_Preferences > View_)?

**Reply by janvinas** [reply]
Disabling scrollbar fadeout doesn't seem to fix the rendering issue (which does indeed look very similar to #4011). Is there any way to disable inertial scrolling on the stable version? If not, will it be implemented in the future? If it will, I can just wait for the next update.

**Reply by danemtsov** [reply]
The fix for #4011 has been merged and will be available with the next nightly build.
